ios/FileAccess.swift

```diff
@@ -179,6 +179,18 @@ class FileAccess: NSObject {
         downloadTask.resume()
     }
 
+    @objc(getAppGroupDir:withResolver:withRejecter:)
+    func getAppGroupDir(groupName: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void {
+        let groupURL = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: groupName)
+
+        if (groupURL == nil) {
+            reject("ERR", "Could not resolve app group directory. The group name '\(groupName)' is invalid.", nil)
+            return
+        }
+
+        resolve(groupURL?.path)
```

Comment:
Could you use the `guard let` assignment to unrap the `nil`?  (See other examples in this file, or search swift docs for the `guard` statement.)

Reply:
Sure, I can do that!